app/javascript/packs/utils/capitalize.js

```diff
@@ -0,0 +1,5 @@
+const capitalize = (str) => {
```

Comment:
str = str.replace("_", " ");

Reply:
You know you want to write a test for this, too. /duck

Reply:
hahahaha. it's capitalize not pretty title.

Reply:
I don't write tests.

Reply:
Do you also not look at the page locally after you add a bunch of react components?
![burn](https://media2.giphy.com/media/3oEhn8TtPQ4MLbFUre/giphy.gif)

Reply:
?

Reply:
Sigh